src/containers/Transactions/simpleTab.scss

```diff
@@ -87,5 +87,30 @@ $subdued-color: $gray-400;
         @include bold;
       }
     }
+
+    .attestation {
+      margin: 16px 0px;
+      background: rgba(34 37 43 / 50%);
+      gap: 15px;
+
+      .attestation-title {
+        display: block;
+        flex-direction: row;
+        padding: 16px 0 0 16px;
+        color: $gray-400;
+        font-size: 14px;
+        font-weight: 600;
+        text-transform: uppercase;
+      }
+
+      .row {
+        padding: 15px 44px 12px 16px;
```

Comment:
The padding here seems strange.  Should it be `16px 44px 12px 16px`?

Reply:
That's what the Figma file had. I'll double check.